packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.0+8
+
+* Fixed a memory leak error For iOS.
+=======
```

Comment:
```suggestion

```

Reply:
This should actually be a blank line, not removed completely.